src/ui/dialog-vkbasalt.ui

```diff
@@ -0,0 +1,819 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<!-- Created with Cambalache 0.10.2 -->
+<interface>
+  <!-- interface-name dialog-vkbasalt.ui -->
+  <requires lib="gtk" version="4.0"/>
+  <requires lib="libadwaita" version="1.0"/>
+  <template class="VkBasaltDialog" parent="AdwWindow">
+    <property name="default-width">550</property>
+    <property name="modal">True</property>
+    <property name="title" translatable="yes">vkBasalt Settings</property>
```

Comment:
Add a comment for translators `... name="title" translatable="yes" comments="vkBasalt is a utility name">...` or something of the sort so they don't try to look it up in a dictionary.